packages/insomnia-inso/src/commands/run-tests.js

```diff
@@ -19,38 +18,65 @@ export const TestReporterEnum = {
 export type RunTestsOptions = GlobalOptions & {
   env?: string,
   reporter: $Keys<typeof TestReporterEnum>,
+  reporterOptions?: Array<string>,
   bail?: boolean,
   keepFile?: boolean,
   testNamePattern?: string,
 };
 
-function validateOptions({ reporter }: RunTestsOptions): boolean {
-  if (reporter && !TestReporterEnum[reporter]) {
-    const reporterTypes = Object.keys(TestReporterEnum).join(', ');
-    logger.fatal(`Reporter "${reporter}" not unrecognized. Options are [${reporterTypes}].`);
-    return false;
+export function isReporterFailure(reporter: string, err: string): void {
+  if (err.includes('invalid reporter')) {
+    logger.fatal(`The following reporter \`${reporter}\` was not found!`);
+  } else {
+    logger.fatal(`An unknown error occurred: ${err}`);
```

Comment:
Because this is a complete failure we don't necessarily need to prefix it with anything.

https://github.com/Kong/insomnia/blob/eff9cda1b2862863de0ce832dee48b00e454bcea/packages/insomnia-inso/src/commands/lint-specification.js#L43

```suggestion
    logger.fatal(err);
```